spec/params/params.spec.ts

```diff
@@ -172,6 +195,18 @@ describe("Params value extraction", () => {
     expect(trueParam.cmp("<=", false).value()).to.be.false;
   });
 
+  it("can test list params for equality but not < or >", () => {
+    const p1 = params.defineList("LIST");
+    const p2 = params.defineList("ESCAPED_LIST");
+
+    expect(p1.equals(p1).value()).to.be.true;
+    expect(p1.notEquals(p1).value()).to.be.false;
+    expect(p1.equals(p2).value()).to.be.false;
+    expect(p1.notEquals(p2).value()).to.be.true;
+
+    expect(() => p1.greaterThan(p1).value()).to.throw;
```

Comment:
This is why I wish we had done operators in the type system with strong types for each expression type =/